Packs/IOCParser/Integrations/IOCParser/IOCParser.py

```diff
@@ -18,87 +16,94 @@
 
 KEYS = ['ASN', 'BITCOIN_ADDRESS', 'CVE', 'DOMAIN', 'EMAIL', 'FILE_HASH_MD5', 'FILE_HASH_SHA1',
         'FILE_HASH_SHA256', 'IPV4', 'IPV6', 'MAC_ADDRESS', 'MITRE_ATT&CK', 'URL', 'YARA_RULE']
+URL = 'https://api.iocparser.com'
 
 
 ''' CLIENT CLASS '''
 
 
 class Client(BaseClient):
-    """
-    """
 
     @logger
-    def __init__(self, headers, verify=False, proxy=False):
-        url = 'https://api.iocparser.com'
-        super().__init__(url, headers=headers, verify=verify, proxy=proxy)
+    def __init__(self, verify=False, proxy=False):
+        url = URL
+        super().__init__(url, verify=verify, proxy=proxy)
 
-    def ioc_from_url(self, url: str) -> Dict[str, Any]:
+    def ioc_from_url(self, url: Optional[str]) -> Dict[str, Any]:
         """
-        Extracts all IOCs from a URL
-
-        :type url: ``str``
-        :param url: The URL from which the IOCs will be extracted
+        Extracts all IOCs from a URL.
+        Args:
+            url: The URL from which the IOCs will be extracted
 
-        :return: The HTTP response returned by the API
-        :rtype: ``Dict[str, Any]``
+        Returns:
+            The HTTP response returned by the API
         """
 
         data = {'url': url, 'public': False}
         return self._http_request(
             method='POST',
             url_suffix='/url',
             headers={'Content-Type': 'application/json'},
+            # The API requires the data to be a string in json format
             data=json.dumps(data),
             return_empty_response=True
         )
 
-    def ioc_from_text(self, text: str) -> Dict[str, Any]:
+    def ioc_from_json_text(self, text: Optional[str]) -> Dict[str, Any]:
         """
-        Extracts all IOCs from a JSON text
-
-        :type text: ``str``
-        :param text: The JSON from which the IOCs will be extracted
+        Extracts all IOCs from a JSON text.
+        Args:
+            text: The JSON from which the IOCs will be extracted
 
-        :return: The HTTP response returned by the API
-        :rtype: ``Dict[str, Any]``
+        Returns:
+            The HTTP response returned by the API
```

Comment:
```suggestion
            Dict[str, Any]. The HTTP response returned by the API
```

Fix in all relevant places.